docs/DeveloperGuide.md

```diff
@@ -52,17 +52,23 @@ to a more efficient and organised way of managing the volunteers’ and elderly
-| `*`              | single administrator of a small NPOs | see how long a volunteer has been with the program                    | assess their experience                       |
-| `*`              | single administrator of a small NPOs | track the befriending history of a volunteer                          | audit past involvements easily                |
+| Priority         | As a …​                            | I want to …​                                                          | So that I can…​                                                                 |
+|------------------|------------------------------------|-----------------------------------------------------------------------|---------------------------------------------------------------------------------|
+| `* * *`          | single administrator of small NPOs | list volunteers                                                       | see all the volunteers and their information                                    |
+| `* * *`          | single administrator of small NPOs | add volunteer to the list of volunteers                               | include new volunteers in the application                                       |
+| `* *`            | single administrator of small NPOs | delete volunteers                                                     | remove volunteers that have left                                                |
+| `* *`            | single administrator of small NPOs | update volunteers & volunteer information                             | keep the volunteer information up-to-date                                       |
+| `* * `           | single administrator of small NPOs | search for particular volunteers by keywords                          | quickly see the volunteer's details                                             |
+| `*`              | single administrator of small NPOs | manage volunteers by region                                           | arrange the volunteers such that they can conveniently reach out to the elderly |
+| `*`              | single administrator of small NPOs | manage the volunteers' available dates and time                       | efficiently find volunteers available for activities                            |
+| `* *`            | single administrator of small NPOs | view nursing / medical courses that volunteers have taken in the past | pair an elderly with a more suitable volunteer                                  |
+| `*`              | single administrator of small NPOs | see how long a volunteer has been with the program                    | assess their experience                                                         |
+| `*`              | single administrator of small NPOs | track the befriending history of a volunteer                          | audit past involvements easily                                                  |
```

Comment:
For add volunteers and delete volunteers, may not need to write benefits? Since the benefits is already quite obvious
And do you consider also include the sub-points in the draft user stories? Such as "filter volunteers by tags" etc

Reply:
I wrote the benefits to reduce ambiguity and make it more consistent.

Writing in the sub points is a good idea though, I'll go ahead and do that.